packages/workbox-build/src/lib/bundle.js

```diff
@@ -0,0 +1,109 @@
+/*
+  Copyright 2018 Google LLC
```

Comment:
Super nit: I think for new files we should use the current year, 2019 (here and for other files as well).